### PR TITLE
Add keycloak-gatekeeper tutorial

### PR DESCRIPTION
**Pull request by vandersonmota**
This is a simplified flow based on the internal doc I initially created. I added "vanilla" k8s yamls for this example.

### CODE REVIEW

File: pages/04.tutorials/27.setup-jwt-authentication-with-keycloak-gatekeeper/default.en.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+First of all you need one **client_id** (and the **client_secret**) from your **keycloak realm** to be
+used by **gatekeeper**. You can find more about this in the [keycloak tutorial](../06.external-authentication/default.en.md).
+**Access type** should be `confidential`. To make easier to follow let's say your
+**client_id** here is `gatekeeper`.
+
+Then, you need another pair of **client_id** and **client_secret** to be used by your API client.
+This one will have the following differences from the first one when adding the Client:
+
+- **client_id** here can be anything, like `my_client_app`
+- On **Advanced settings** it is advisable to increase the **Access token lifespan**,
```
Comment (stefanandres) [inline pages/04.tutorials/27.setup-jwt-authentication-with-keycloak-gatekeeper/default.en.md]: why and to what value shall it be increased?

File: pages/04.tutorials/27.setup-jwt-authentication-with-keycloak-gatekeeper/default.en.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+1. After clicking the **Client Scopes** menu on the sidebar, create one. (for ex: `api`)
+2. Then within `api` settings page, go to **Mappers** tab and create one with the following configuration:
+    - **Name** can be anything, let's say `api-audience`
+    - **Mapper type** as `Audience`
+    - **Included Client Audience** to the first **client_id** you added (`gatekeeper`).
+3. Go back to the **Clients** menu on the sidebar then to do the following for both clients you created at the **Client Scopes** tab in their settings page:
+    - Add the **Client scope** you created (`api`) to the assigned default client scopes.
+
+Finally, you'll need to add an user to be used by your API's client applications.
```
Comment (stefanandres) [inline pages/04.tutorials/27.setup-jwt-authentication-with-keycloak-gatekeeper/default.en.md]: add an user to where? the realm?

File: pages/04.tutorials/27.setup-jwt-authentication-with-keycloak-gatekeeper/default.en.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+spec:
+  ports:
+  - name: http
+    port: 80
+    protocol: TCP
+    targetPort: service
+  selector:
+    app: myservice
+  type: ClusterIP
+```
```
Comment (stefanandres) [inline pages/04.tutorials/27.setup-jwt-authentication-with-keycloak-gatekeeper/default.en.md:150]: can you add an example on how authentication then looks like / what the requests/responses are? e.g. some curl or some other simple app

**Review by stefanandres — CHANGES_REQUESTED** [review]

File: pages/04.tutorials/27.setup-jwt-authentication-with-keycloak-gatekeeper/default.en.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+taxonomy:
+    tag:
+        - kubernetes
+        - Keycloak
+        - JWT
+---
+
+## Overview
+
+For applications that are API-only, setting up authentication with JWT tokens is a reasonable choice,
```
Comment (stefanandres) [inline pages/04.tutorials/27.setup-jwt-authentication-with-keycloak-gatekeeper/default.en.md]: please add an example for what scenario where you might use this to get a broader overview for whom this tutorial is for

**Comment by vandersonmota** [comment]
@stefanandres I think I made the changes you requested, plus changed a few other things for clarity.

**Review by stefanandres — APPROVED** [review]

**Pull request merged**